apps/native-component-list/src/navigation/MainTabNavigator.tsx

```diff
@@ -52,14 +57,15 @@ export default function MainTabbedNavigator(props: any) {
           borderTopWidth: StyleSheet.hairlineWidth,
           borderTopColor: Colors.tabIconDefault,
         }}
+        // bottom-tabs props
         tabBarOptions={{
           style: {
             backgroundColor: Colors.tabBar,
           },
           activeTintColor: Colors.tabIconSelected,
           inactiveTintColor: Colors.tabIconDefault,
         }}>
-        {Object.keys(Screens).map(name => (
+        {(Object.keys(Screens) as (keyof typeof Screens)[]).map(name => (
```

Comment:
I still believe that using
```suggestion
        {Object.entries(Screens).map(([name, Component]) => (
```
would solve TS errors and wouldn't force us to use this nasty casting 😉